alerts/cluster-network-operator/NorthboundStaleAlert

```diff
@@ -0,0 +1,22 @@
+# Northbound Stale Alert
+
+## Meaning
+
+The alert `NorthboundStale` is triggered when ovn-northd has gone longer than 2 minutes 
```

Comment:
Its not 2 minutes. The alert becomes active ~12 minutes after the last timestamp write to SB DB.
2 minutes for the alert to become "active" and then theres a "For" for 10 minutes, so it must stay active for 10 minutes.

Reply:
I am thinking maybe we should merge the two periods? It sounds to me that give it 12 minutes without timestamp update/ovnkube-master correct response is a bit problematic. 
How about if we set the expr to `time() - max(ovnkube_master_nb_e2e_timestamp) > 300(or 600 if it is acceptable)` and fire it immediately?

Reply:
Yeah. Its awkward as hell. Can you make that change for 600 bo?

Reply:
https://github.com/openshift/cluster-network-operator/pull/1377 here it is

Reply:
thanks bo